src/node_contextify.cc

```diff
@@ -394,7 +394,8 @@ class ContextifyContext {
     Local<String> key = property->ToString(context).ToLocalChecked();
     Local<Object> sandbox = ctx->sandbox();
 
-    if (sandbox->HasOwnProperty(context, key).FromJust()) {
+    Maybe<bool> has = sandbox->HasOwnProperty(context, key);
+    if (has.IsJust() && has.FromJust()) {
```

Comment:
Not wrong but `if (has.FromMaybe(false)) {` is more common in our code base.

Reply:
Thanks. Changed.